packages/terra-application-navigation/src/tabs/_Tab.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+
+import TabCount from './_TabCount';
+import { enableFocusStyles, disableFocusStyles, generateKeyDownSelection } from '../utils/helpers';
+
+import styles from './Tab.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The display text for the tab.
+   */
+  text: PropTypes.string.isRequired,
+  /**
+   * The selection callback for the tab.
+   */
+  onTabSelect: PropTypes.func,
+  /**
+   * The number of notifications to be displayed for the tab.
+   */
+  notificationCount: PropTypes.number,
+  /**
+   * Boolean indicating whether or not the Tab should render as active.
+   */
+  isActive: PropTypes.bool,
+  /**
+   * Boolean indicating whether or not the tab is a placeholder for height calculation.
+   */
+  isPlaceholder: PropTypes.bool,
+  /**
+   * Ref object that will be updated with a reference to the Tab's root element.
+   */
+  tabRef: PropTypes.shape({ current: PropTypes.instanceOf(Element) }),
+  /**
+   * Boolean indicating whether or not the Tab should account for count spacing.
+   */
+  hasCount: PropTypes.bool,
+  /**
+   * Render prop for dynamic tab contents.
+   */
+  render: PropTypes.func,
+};
+
+const Tab = ({
+  text,
+  hasCount,
+  isActive,
+  isPlaceholder,
+  tabRef,
+  render,
+  notificationCount,
+  onTabSelect,
+}) => {
+  function renderTabContent() {
+    if (render) {
+      return render({
+        text,
+        hasCount,
+        isActive,
+        notificationCount,
+      });
+    }
+
+    return (
+      <span className={cx('tab-inner')}>
+        <span className={cx('tab-label')}>{text}</span>
+        {notificationCount > 0 && <span className={cx('tab-count')}><TabCount value={notificationCount} /></span>}
+      </span>
+    );
+  }
+
+  return (
+    <div
+      type="button"
```

Comment:
Something seems odd here, we have a div with type="button" and role="link". Need to decide if we want to treat this a button or a link. cc/ @scottwilmarth

Reply:
Talked with @dkasper-was-taken. They will be meeting with @scottwilmarth to discuss accessibility of the component today.